src/Options/index.js

```diff
@@ -87,7 +87,7 @@ export interface ParseServerOptions {
   verbose: ?boolean;
   /* Sets the level for logs */
   logLevel: ?string;
-  /* (Optional) Overrides the log levels used internally by Parse Server to log events.
+  /* Options for customizing log levels for specific events
```

Comment:
Do not modify this and see the existing feedback regarding adding nee Parse Server options